support-frontend/app/controllers/Application.scala

```diff
@@ -112,7 +112,7 @@ class Application(
     val guestAccountCreationToken = request.flash.get("guestAccountCreationToken")
 
     implicit val settings: AllSettings = settingsProvider.getAllSettings()
-    request.user.traverse[Attempt, IdUser](identityService.getUser(_)).fold(
+    request.user.traverse[Attempt, IdUser](user => identityService.getUser(user.user)).fold(
```

Comment:
syntax is now `user.user` since I've removed an implicit def converting one type of a user to another